src/main/java/net/sf/jabref/logic/importer/fetcher/CrossRef.java

```diff
@@ -114,16 +111,6 @@ private static boolean checkValidity(BibEntry entry, JSONArray result) {
         }
     }
 
-    private static String removeLaTeX(String text) {
-        String result;
-        // remove braces
-        result = new RemoveBracesFormatter().format(text);
```

Comment:
I re-added it, but I don't know if its important. The latex-free field and the removeBraces are only used in the validation method, in the `findDOI` method the normal field is used.
Maybe someone has to look over it.